setup.cfg

```diff
@@ -18,3 +18,4 @@ packages = bring_api
 python_requires = >=3.8
 install_requires =
     aiohttp
+include_package_data = True

```

Comment:
does that make sure that the json files will be packaged?

Reply:
it makes sure all files within package folder bring_api are included. Optionally a MANIFEST.in could be included, to explicitly name file to include. Tested it locally with `pip install --editable .` and it included all json files. successfully.